### PR TITLE
feature/edit_btn > edit button added

### PR DESCRIPTION
**Pull request by jeesunikim**
Edit button added:
edit_button_example

- Please add "link" in frontmatter if we want to link it to a relevant github link

### CODE REVIEW

File: content/api/introduction/index.mdx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Horizon API Reference
 order: 0
+link: https://github.com/stellar/new-docs/blob/master/content/api/introduction/index.mdx
```
Comment (jeesunikim) [inline content/api/introduction/index.mdx]: Please add "link" in frontmatter if we want to link it to a relevant github link @kylemccollom

Comment (kylemccollom) [reply content/api/introduction/index.mdx]: can we set up this edit button to link to the relevant `.mdx` file? for example, the edit button on https://stellar-documentation.netlify.com/api/resources/ledgers/object/ would link to https://github.com/stellar/new-docs/blob/master/content/api/resources/ledgers/object.mdx. so each page's edit button would auto link to the file that contains the contents from which the page was built.

Comment (jeesunikim) [reply content/api/introduction/index.mdx]: smart

Comment (jeesunikim) [reply content/api/introduction/index.mdx]: Updated

**Pull request merged**